src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -171,6 +174,10 @@ const ApplicationLauncher = _.flow(
         .Runtimes
         .fileSyncing(googleProject, runtime.runtimeName)
         .setStorageLinks(localBaseDirectory, localSafeModeBaseDirectory, cloudStorageDirectory, `.*\\.Rmd`)
```

Comment:
Not sure if the last argument here is a regular expression or a glob, but could this handle something like `.*\\.R(md)?` instead of requiring two requests.